behaviours/rant_patrol.py

```diff
@@ -16,14 +17,15 @@ async def main(event: hikari.GuildMessageCreateEvent):
     in_channel = event.channel_id == int(rant_channel_id)
     content = event.content
     is_a_rant = RANT_REGEX.match(content)
+    is_stem_valid = STEM_REG.match(content)
```

Comment:
If you're trying to do images, you should also hedge this by checking for an attachment.